tests/scripts/visual-coverage.sh

```diff
@@ -6,9 +6,11 @@
 # generate coverage from storybook test-runner
 test-storybook --coverage
 
-# kill the locally served Storybook
-kill -9 $JOB_ID
-
+# kill the locally served Storybook started by the test
+if [[ -z $process ]]
```

Comment:
will this always kill the local storybook, even if it wasn't started as part of this script?

Reply:
This will kill the local storybook only if it is started by the test and will leave the storybook started before the test running.